### PR TITLE
Add AFUP Day 2023

### PR DESCRIPTION
**Pull request by kpn13**
A PHP event.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,6 +146,7 @@ This tech conferences agenda list can be seen in https://developers.events/
 
 * ??: [Codecamp_Festival](https://codecamp.ro/the-festival-book-now/) - Bucharest, Romania
 * 3-4: [DevOps Days Zurich](https://devopsdays.org/events/2023-zurich/welcome/) - Zurich (Switzerland)
+* 12: [AFUP Day](https://event.afup.org/) <img alt="CFP AFUP Day 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2014-November-2022&color=green"></a>
```
Comment (scraly) [inline README.md:149]:
```suggestion
* 12: [AFUP Day](https://event.afup.org/) Lille & Lyon (France) <img alt="CFP AFUP Day 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2014-November-2022&color=green"></a>
```

**Review by scraly — CHANGES_REQUESTED** [review]
thanks
just a suggestion to display location

**Comment by scraly** [comment]
merci :)

**Comment by kpn13** [comment]
Merci à toi :)

**Pull request merged**